app/views/pages/home.html.erb

```diff
@@ -7,7 +7,7 @@
       <div class="col-12 col-sm-6 col-md-4 mr-6 bookings-list-item">
         <div class="card-image">
       </div>
-      <div class="card-body p-3">
+      <div class="card-body p-3 shadow">
```

Comment:
If you want the shadow to be on the card then the shadow should be in the parent class. Otherwise OK

Reply:
The shadow looks strange on the parent class, looks better here.